### ISSUE TITLE
Engine stalls when going into gear, even when using clutch

### ISSUE THREAD

**Issue by sheepsushis** [ange-yaghi/engine-sim#17]

**Comment by PreloadedAssets** [comment]
If you're using the base engine then the reason is it just doesn't produce enough HP.
Mess with the files for the engines (they're modifiable without compiling/building the project) under `build_1_3a\assets\partlibrary\engines` The default engine is the `kohler_ch750.mr` file.

**Comment by naikrovek** [comment]
this is an issue about the default engine, then.  it should be an engine which can power the virtual vehicle it's in.

**Comment by ExampleWasTaken** [comment]
> this is an issue about the default engine, then. it should be an engine which can power the virtual vehicle it's in.

I think you're forgetting that this is a personal project, so you can't expect anything from the developer. He's doing it for himself. Not for you ;)

**Comment by naikrovek** [comment]
> I think you're forgetting that this is a personal project, so you can't expect anything from the developer. He's doing it for himself. Not for you ;)

I think you're forgetting that I am making zero demands of the author of the software, nor am I asking for the author to do anything at all; I am asking the person who opened this issue to change the issue to reflect what the actual problem is: the default engine isn't powerful enough to exercise the simulator.

;)

**Comment by DogeSkii** [comment]
discord exists

**Comment by naikrovek** [comment]
> discord exists

yes, and I am there, if you would care to look.  But you can't edit this issue in Discord, can you?

I'm talking about editing *this GitHub Issue* to correctly reflect what the problem actually is.  that's all.

**Comment by DogeSkii** [comment]
> > discord exists
> 
> 
> 
> yes, and I am there, if you would care to look.  But you can't edit this issue in Discord, can you?
> 
> 
> 
> I'm talking about editing *this GitHub Issue* to correctly reflect what the problem actually is.  that's all.  

yeah but the GitHub issues are meant for bugs and actual issues, not user errror, legit have support channels in discord

**Comment by naikrovek** [comment]
> yeah but the GitHub issues are meant for bugs and actual issues

Yes, correct.  The actual issue is that the default simulated engine is not powerful enough to drive the simulated car.  That is an issue.  A person downloading this software won't be able to fully use it without taking it apart and/or joining Discord and asking about it.  That is a problem.  That problem is why this issue was filed, to create a record of that problem.  However, this github issue is worded poorly, and it is not immediately clear about what the actual software issue is.  I am asking the creator of this issue to clarify their intent and modify the issue to indicate what the actual problem is, that the default engine selected when you run this software (the V-Twin engine) is not powerful enough to drive the car simulated in this software.

**Comment by sheepsushis** [comment]
I have added a description to the question. Sorry for the confusion.

**Comment by sanchez** [comment]
Have you given it enough throttle before you let out the clutch?

**Comment by arturs4junction** [comment]
Has anyone managed to tweak the default `kohler_ch750.mr` engine to give enough power to get into gear?
If you have, could you please share the configuration. I am not a mechanic, I have no idea how any of the parameters might influence the power.

**Comment by macluxHD** [comment]
In this commit: bfed02dd5fafffb9a7162afb87e8f2f81b7e2e38 ange added a new engine. Just download the new test.mr file and the i5.mr file and replace/add them in your assets folder. This new engine is powerful enough to change gears.

This is not part of the prebuilt package but I assume that they are going to update it sometime.

**Comment by arturs4junction** [comment]
> In commit [bfed02d](https://github.com/ange-yaghi/engine-sim/commit/bfed02dd5fafffb9a7162afb87e8f2f81b7e2e38) ange added a new engine. Just download the new test.mr file and the i5.mr file and replace/add them in your assets folder. This engine is powerful enough to change gears.
> 
> This is not part of the prebuilt package but I assume that they are going to update it sometime soon.

Amazing find, thank you very much!

**Comment by Ashdemai** [comment]
What is even the bujtton for the throttle?

**Comment by JacksonBailey** [comment]
> > > discord exists
> > 
> > 
> > yes, and I am there, if you would care to look.  But you can't edit this issue in Discord, can you?
> > I'm talking about editing _this GitHub Issue_ to correctly reflect what the problem actually is.  that's all.
> 
> yeah but the GitHub issues are meant for bugs and actual issues, not user errror, legit have support channels in discord

Please consider that it's not always immediately obvious if something is a bug or user error.
